Cubical/Categories/Limits/Pullback.agda

```diff
@@ -26,37 +28,41 @@ module _ {C : Category ℓ ℓ'} where
       s₁ : C [ l , m ]
       s₂ : C [ r , m ]
 
```

Comment:
I'm not sure that it's worth having a separate record for cospans... What do you think @ecavallo ?

Reply:
I think it's fine to have. I think the category parameter should be explicit, though.

Reply:
I wonder if it should be explicit to all definitions in the module or not... Hmm

Reply:
I made it explicit